source/xor.js

```diff
@@ -0,0 +1,27 @@
+import _curry2 from './internal/_curry2';
+
+
+/**
+ * Exclusive disjunction logical operation.
+ * Returns `true` if one of the arguments is `true` and the other is `false`.
+ * Otherwise, it returns `false`.
+ *
+ * @func
+ * @memberOf R
+ * @category Logic
+ * @sig a -> b -> Boolean
+ * @param {Any} a
+ * @param {Any} b
+ * @return {Boolean} true if one of the arguments is truthy and the other is falsy
```

Comment:
As implemented, the actual return type is `Any`. :\

```javascript
> R.xor (0) (0)
0
```

Reply:
Right.  Perhaps `!!(a || b) && !(a && b)` would be appropriate.

Reply:
> !!(...)

I think you spelled `Boolean` wrong 😛

Reply:
Long habit.  But yes, clearly `Boolean(a || b)` is better.

Reply:
@davidchambers Good point, thanks!

@ all: I know there is a long discussion about `and` and `or` returning types. Do you think this should return Any, or be converted to Boolean instead?

Reply:
I think returning a boolean is fine here.  We're not trying to mimic a native boolean xor function here.

Reply:
Okay, I'll change my commit and also add the @see docs to `and` and `or`